HW1 Crystal Resume Webpage.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+<html lang="en">
+<head>
+<style>
+h1 {
+    font-family: "Calibri";
+	text-align: center;
+}
+
+p {
+    font-family: "Calibri";
+	background-color:#999999;
+	color: #FFFFFF;
+	text-align: center;
+
+}
+body {
+    background-image: url("background.jpg");
+	background-size: contain;
```

Comment:
Change this to `background-size: cover;` so that the image is covering the whole page instead of contained by the page, which means the image might be smaller than the content and the the image won't cover the whole background. `Cover` means you it uses the shortest side of the image and makes sure that the image is covering all the content area. however the using `cover` doesn't guarantee showing the complete image. it's usually cut off.